### PR TITLE
Update event fires properly on campaign group changes

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Check when updating a campaign if the group changes, if so update it's fires
* Remove any unfired fires when deleting a group
* Run update fires for campaign in celery
* Don't show inactive groups on campaign read page

### CODE REVIEW

File: temba/campaigns/tasks.py [hunk 1]
```diff
@@ -49,3 +49,19 @@ def update_event_fires(event_id):
 
         # bubble up the exception so sentry sees it
         raise e
+
+@task(track_started=True, name='update_event_fires_for_campaign_task') # pragma: no cover
+def update_event_fires_for_campaign(campaign_id):
+    try:
```
Comment (nicpottier) [inline temba/campaigns/tasks.py]: Should we have a lock so these don't stack up?

**Comment by nicpottier** [comment]
Man, crazy to think this has been there all along. Looks good.

**Comment by norkans7** [comment]
Looks good

File: temba/campaigns/tasks.py (hunk 1 above)
Comment (ericnewcomer) [inline temba/campaigns/tasks.py]: I suppose it'd be good to make sure we don't kick off more than one of these for a given campaign at a time. When I add a lock on campaign, if they create a bunch of edit operations, they will instead stack up with the lock, but they do all need to run eventually, so stacking seems like the correct behavior there.

Comment (nicpottier) [inline temba/campaigns/tasks.py]: Ya, mostly worried about multiple of these running at once.. them blocking for the prior one to complete seems like the best option no?

Comment (ericnewcomer) [inline temba/campaigns/tasks.py]: Agreed. Was just confused by the stack up comment.

Comment (nicpottier) [inline temba/campaigns/tasks.py]: Ya, sorry, my bad.

**Comment by nicpottier** [comment]
:zap:

**Pull request merged**